hack/build_multiarch_clis.sh

```
@@ -9,58 +9,153 @@ set -euo pipefail
+
+cd "$SOURCE_DIR"
+
+# Build JSON array of artifact entries
+artifacts=$(find . -type f | while read -r file; do
+  filename=$(basename "$file")
+
+  # Skip index.json and .sha256 files
+  [[ "$filename" == "$OUTPUT_JSON" || "$filename" == *.sha256 ]] && continue
+
+  # Extract arch and os from path
+  parts=(${file//\// })
```

Shellcheck (0.10.0)

[warning] Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a.

(SC2206)